### ISSUE TITLE
pullcves downloads wrong pages

### ISSUE THREAD

**Issue by mad-ady** [sjvermeu/cvechecker#18]
I've just installed the latest git snapshot and tried to populated the CVE database by running:
``` pullcves pull```

It tries to download xml files from 2002 - 2015, but every file fails the conversion because it's not XML, it is HTML instead. It seems the site changed the structure and now serves HTML documents for the addreses the program queries. A relevant snippet for such a document (from /usr/local/var/cvechecker/cache/nvdcve-2.0-2015.xml) shows the following:
```
adrianp@stingray:~/temp/cvechecker$ file /usr/local/var/cvechecker/cache/nvdcve-2.0-2015.xml
/usr/local/var/cvechecker/cache/nvdcve-2.0-2015.xml: HTML document, ASCII text, with very long lines, with CRLF, LF line terminators
adrianp@stingray:~/temp/cvechecker$ egrep "frame window|redirected to" /usr/local/var/cvechecker/cache/nvdcve-2.0-2015.xml
        <h1>You are viewing this page in an unauthorized frame window.</h1>
        <p>This is a potential security issue, you are being redirected to <a href="http://nvd.nist.gov">http://nvd.nist.gov</a></p>
```
Could you please check if the URL needs updating?

**Comment by sjvermeu** [comment]
Darn they seem to have updated their site code. I'm going to see what the correct URL has to be.

**Comment by sjvermeu** [comment]
Okay seems that the uncompressed downloads are no longer supported. I'm switching to the compressed ones. Allow me to test the stuff a bit and I'll build up a new release.

**Comment by sjvermeu** [comment]
Hi mad-ady, I've released version 3.6 which also includes the fix for this bug.

https://github.com/sjvermeu/cvechecker/wiki
https://github.com/sjvermeu/cvechecker/wiki/Download
